src/node_options-inl.h

```diff
@@ -23,12 +23,14 @@ template <typename Options>
 void OptionsParser<Options>::AddOption(const char* name,
                                        const char* help_text,
                                        bool Options::* field,
-                                       OptionEnvvarSettings env_setting) {
+                                       OptionEnvvarSettings env_setting,
+                                       bool default_is_true) {
   options_.emplace(name,
                    OptionInfo{kBoolean,
                               std::make_shared<SimpleOptionField<bool>>(field),
                               env_setting,
-                              help_text});
+                              help_text,
+                              default_is_true});
```

Comment:
I think I'm doing something wrong here, but was not able to find what.

Even when `AddOption` is called with `default_is_true = true`, the value of `OptionInfo.default_is_true` will always be false.

Reply:
I think you’ll also want to add the field to the relevant `OptionsParser<Options>::Convert` overload here